README.md

```diff
@@ -10,17 +10,35 @@ This GitHub Action (created using typescript) uses [CLI for Microsoft 365](https
 Create a workflow `.yml` file in your `.github/workflows` directory. An [example workflow](#example-workflow---cli-for-microsoft-365-login) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
 
 ### Inputs
-- `ADMIN_USERNAME` : **Required** Username (email address of the admin)
-- `ADMIN_PASSWORD` : **Required** Password of the admin
+- `ADMIN_USERNAME` : Username (email address of the admin)
+- `PASSWORD` : Password for the admin or the certificate
+- `CERTIFICATE_ENCODED` : Base64-encoded string with certificate private key
+- `AAD_APP_ID` : App ID of the Azure AD application to use for certificate authentication
+- `TENANT_ID` : ID or domain (for example "contoso.onmicrosoft.com") of the tenant from which accounts should be able to authenticate
+
+All inputs are optional. But depending of the authentication type chosen, following pair of inputs will be necessary:
+
+- `authType` is `password`: `ADMIN_USERNAME` and `PASSWORD` are required
+- `authType` is `certificate`: at least `CERTIFICATE_ENCODED` and `AAD_APP_ID` are required
+  - Depending on your tenant environment, `TENANT_ID` will be required
```

Comment:
_Depending on your tenant environment_

Can we clarify this?

Reply:
Maybe I could copy / paste what is said on CLI for Microsoft 365 [login](https://pnp.github.io/cli-microsoft365/cmd/login/) documentation page, except the `CLIMICROSOFT365_TENANT` environment variable part? Like this:

```
ID of the tenant from which accounts should be able to authenticate. Use common or organization if the app is multitenant.
```

Reply:
That would be better indeed 👍